contrib/mesos/docs/ha.md

```diff
@@ -66,5 +86,417 @@ The command line parameters that affect the hash calculation are listed below.
 - `--proxy-*`
 - `--static-pods-config`
 
+## Cold-standby
+
+Setting up Kubernetes on Mesos in cold-standby mode is similar to Kubernetes in
+standalone mode described in [Kubernetes HA][1]. However, special attention is
```

Comment:
nit: "However"? It seems to me that the same special attention is needed in both cases - what's the difference between the cold and hot standby cases w/ respect to consistency among flags?